tensorrt_llm/tools/layer_wise_benchmarks/runner.py

```
@@ -339,36 +355,92 @@ def forward_impl(*args, **kwargs):
+        )
+        model_loader = ModelLoader(
+            llm_args=llm_args,
+            mapping=mapping,
+            spec_config=None,
+            sparse_attention_config=None,
+            max_num_tokens=max_num_tokens,
+            max_seq_len=max_seq_len,
+        )
+
+        with self.scaled_from_ctx(scaled_from, mapping), self.skip_unused_layers_ctx(layer_indices):
+            model, moe_load_balancer = model_loader.load(
```

Ruff (0.14.10)

Unpacked variable `moe_load_balancer` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)